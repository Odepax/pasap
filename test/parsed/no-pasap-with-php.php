<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>Pasap Test</title>
	</head>
	<body>
		<header>
			<nav>
				<?php foreach([ "Wow", "Amaze", "Youpi" ] as $a): ?>
					<a href=""><?= $a ?></a>
				<?php endforeach; ?>
			</nav>
		</header>
		<main>
			<h1>The list</h1>
			<ul>
				<?php foreach([ "Wow", "Amaze", "Youpi" ] as $a): ?>
					<li><?= $a ?></li>
				<?php endforeach; ?>
			</ul>
		</main>
	</body>
</html>
