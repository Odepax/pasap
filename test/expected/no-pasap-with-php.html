<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Pasap Test</title>
	</head>
	<body>
		<header>
			<nav>
				<a href="">Wow</a>
				<a href="">Amaze</a>
				<a href="">Youpi</a>
			</nav>
		</header>
		<main>
			<h1>The list</h1>
			<ul>
				<li>Wow</li>
				<li>Amaze</li>
				<li>Youpi</li>
			</ul>
		</main>
	</body>
</html>
